modules/core/src/test/java/org/apache/ignite/internal/processors/database/DataRegionMetricsSelfTest.java

```diff
@@ -74,9 +74,11 @@
 
         IgniteConfiguration cfg = new IgniteConfiguration().setMetricExporterSpi(new NoopMetricExporterSpi());
 
-        memMetrics = new DataRegionMetricsImpl(plcCfg,
-            new GridMetricManager(new GridTestKernalContext(new GridTestLog4jLogger(), cfg)),
-            NO_OP_METRICS);
+        GridTestKernalContext ctx = new GridTestKernalContext(new GridTestLog4jLogger(), cfg);
+
+        ctx.add(new GridMetricManager(ctx));
+
+        memMetrics = new DataRegionMetricsImpl(plcCfg, ctx.metric(), ctx.performanceStatistics(), NO_OP_METRICS);
```

Comment:
Should we add `PerformanceStatisticsProcessor` to the `ctx` first?
Is seems `ctx.performanceStatistics()` will always return `null` here.

Reply:
done